ref/general/about-open-liberty.adoc

```diff
@@ -48,20 +51,35 @@ Open Liberty offers full compatibility between product versions so that your exi
 
 Open Liberty can be containerized and deployed to any Kubernetes cloud. Check out these guides to learn how you can easily deploy to link:/guides/okd.html[OKD], link:/guides/cloud-openshift.html[OpenShift], link:/guides/cloud-azure.html[Azure Kubernetes Service], link:/guides/cloud-aws.html[Amazon Web Services], link:/guides/cloud-ibm.html[IBM Kubernetes Service], and more.
 
+=== Protect applications and servers
+Open Liberty supports application security with a robust set of security features.
+During development and testing, Open Liberty's built-in link:/docs/ref/general/#basic-registry.html[basic user registry and the QuickStart security element] offer easily configurable security frameworks to test access to your application in a lightweight, self-contained manner.
+The `securityUtility` supports plain text encryption, so you can easily secure user passwords with hash encoding.
+
+After deployment, the link:/docs/ref/feature/#appSecurity.html[Application Security] feature provides the core capability for the Jakarta EE Security API, while the link:/docs/ref/feature/#transportSecurity.html[Transport Security] feature enables secure TLS connections by using HTTPS.
+The link:/docs/ref/feature/#ldapRegistry.html[LDAP User Registry] feature supports authentication and authorization from external LDAP registries.
```

Comment:
"The link:/docs/ref/feature/#ldapRegistry.html[LDAP User Registry] feature supports authentication and authorization from external LDAP registries."
In this sentence, instead of "authentication and authorization from external LDAP registries." should it be "authentication and authorization with external LDAP registries.", or "authentication and authorization to external LDAP registries." ?